common/src/test/java/com/revenuecat/purchases/backend_integration_tests/BaseBackendIntegrationTest.kt

```diff
@@ -87,14 +96,12 @@ abstract class BaseBackendIntegrationTest {
             every { edit() } returns sharedPreferencesEditor
         }
         eTagManager = ETagManager(sharedPreferences)
-        signingManager = SigningManager(SignatureVerificationMode.Disabled, appConfig, apiKey())
+        signingManager = spyk(SigningManager(signatureVerificationMode, appConfig, apiKey()))
```

Comment:
Here I changed to a spy so we can see and verify the calls more easily, but it's using the actual implementation. Since there is no way to know that the offerings/product-entitlement mapping calls were verified or not, this made easier to make sure we were verifying those endpoints

Reply:
> Since there is no way to know that the offerings/product-entitlement mapping calls were verified or not

I checked that by using the "fake invalid signature" thing. Did you add a way to do that in Android too?

Reply:
Well, I meant in the "success" case, we should make sure we are verifying the signature. But right now there is no way to detect whether a `SUCCESS` or a `NOT_REQUESTED` happened. We could maybe if we could verify the logs, but I haven't added that in Android yet. Until then, this makes sure we are at least calling the `signingManager.verify` method, so it shouldn't be `NOT_REQUESTED`.